src/Security/Authorization/Core/ref/Microsoft.AspNetCore.Authorization.netcoreapp3.0.cs

```diff
@@ -160,6 +162,7 @@ public partial interface IAuthorizationPolicyProvider
     {
         System.Threading.Tasks.Task<Microsoft.AspNetCore.Authorization.AuthorizationPolicy> GetDefaultPolicyAsync();
         System.Threading.Tasks.Task<Microsoft.AspNetCore.Authorization.AuthorizationPolicy> GetPolicyAsync(string policyName);
+        System.Threading.Tasks.Task<Microsoft.AspNetCore.Authorization.AuthorizationPolicy> GetFallbackPolicyAsync();
```

Comment:
Are we really just making breaking changes to this interface? Don't people implement it?

Reply:
C#8 default interface methods?

Reply:
Yeah but we already broke it by adding RequiredPolicy and removing it was going to break it too, is renaming it worse/better?

Reply:
There certainly are some people implementing this (I know identity server folks were, but it should be fairly uncommon)  When we regressed a major scenario in 2.x with policy providers we only got a handful of reports

Reply:
> Yeah but we already broke it by adding RequiredPolicy and removing it was going to break it too, is renaming it worse/better?

RequiredPolicy was added after 2.2 - https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authorization.iauthorizationpolicyprovider?view=aspnetcore-2.2

Adding it and removing it during 3.0 dev had no effect outside of preview users.

Reply:
@JamesNK I think the concern is that adding stuff to the interface is a breaking change and we could consider using something like downcasting to a type that apps support for this new feature instead of introducing breaking changes.